src/index.tsx

```diff
@@ -26,5 +30,25 @@ export function ReactAccordion(props: ReactAccordionProps) {
   );
 }
 
+export function SingleItemOpenAccordion(props: { items: Item[] }) {
```

Comment:
What if someone passes 2 items initially with the open prop?

Reply:
I would think the last one should work, and the others should be ignored?

Reply:
Updated.
What do you think?
Best way to do it?

Reply:
Ahh cool. I like the reverse!